client/src/main/java/org/asynchttpclient/netty/channel/ChannelManager.java

```diff
@@ -488,4 +493,28 @@ public ChannelPool getChannelPool() {
     public EventLoopGroup getEventLoopGroup() {
         return eventLoopGroup;
     }
+
+    public ClientStats getClientStats() {
```

Comment:
The implementation of this method does not make me happy, but I really don't see a better way to do this with Java 8 `Stream`s. If you think there's a better approach, let me know.